.devops/nix/package-gguf-py.nix

```diff
@@ -0,0 +1,22 @@
+{
+  lib,
+  llamaVersion,
+  numpy,
+  poetry-core,
+  buildPythonPackage,
+}@inputs:
+
+buildPythonPackage {
+  pname = "gguf";
+  version = llamaVersion;
+  pyproject = true;
+  nativeBuildInputs = [ poetry-core ];
+  propagatedBuildInputs = [ numpy ];
+  src = lib.cleanSource ../../gguf-py;
+  doCheck = false;
```

Comment:
Looks like the tests are currently a no-op, but we should enable them and add `pytestCheckHook` in case they are enabled later. We should also add `pythonImportsCheck`